lib/vsc/install/shared_setup.py

```diff
@@ -156,6 +165,88 @@ def remove_extra_bdist_rpm_files(pkgs=None):
 
     return res
 
+class vsc_sdist(sdist):
```

Comment:
class name should be CamelCase, so `VscSdist`? or do you want to stick to the setuptools style of class names?